src/util.ts

```diff
@@ -13,6 +13,15 @@ export function removeTrailingSlash(path: string) {
 	return path.replace(/[/\\]+$/, '');
 }
 
+/** Remove the subpage segment of a URL string */
+export function removeSubpageSegment(path:string) {
+	// Include new pages with subpages as part of this if statement.
+	if (path.includes('install/' || 'deploy/')) {
```

Comment:
This will never test if the path includes `deploy/`. Maybe do it with a regular expression?

But the fact it’s still working makes we wonder if we even need this utility function 🤔 

```suggestion
	if (/(?:install|deploy)\//.test(path)) {
```

Reply:
lol, I don't even know why I tried to use "OR" inside `.includes()`. 
The reason why it still works is that I changed `.endsWith()` to `.includes()` here:

```jsx
<a href={`${Astro.site.pathname}${lang}/${slug}/`} aria-current={`${currentPageMatch.includes(removeSubpageSegment(slug)) ? 'page' : 'false'}`}>
```

Deploy subpages, like `en/guides/deploy/netlify` includes the slug `guides/deploy` but since the install parent page is `install/auto` we need the utility function. We can completely remove the utility function if we change `install/auto` to only `install`, how do feel about this?

Also, I just found that I was using this utility function in places I didn't even needed it :sweat_smile:

Reply:
I think we can do something similar to deploy as well: change the main page to `index.md` and place it inside the `deploy` folder (Maybe it can be kinda weird to have multiples `index.md` pages, so we can just leave `deploy.md` and `install.md` outside its subpages folder)

Reply:
> We can completely remove the utility function if we change install/auto to only install, how do you feel about this?

Actually, this opens the possibility for future bugs in the current state of things. We need to know it's a URL segment by including `/` as part of `currentPageMatch`, so we can use `includes()` without worrying about matching stuff that shouldn't match. We would still change the automatic installation page to be only `install`, but `currentPageMatch` would not remove the trailing slash, so we are sure it's an entire segment of the URL and not something like `guides/install-integrations`. This way we can just compare the URL fragment and know it is a subpage without a hardcoded utility function mentioning which pages have subpages.

I have this new solution staged and working as intended, I just want to be sure if I can change `install/auto` to `install` just to avoid committing these changes and then having to rollback :sweat_smile:.